front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorCurves.tsx

```diff
@@ -6,7 +6,7 @@ import { isEmpty } from 'lodash';
 import type {
   EffortCurves,
   RollingStock,
-  RollingStockComfortType,
+  Comfort,
 } from 'common/api/osrdEditoastApi';
```

Comment:
To prevent CI to fail because of eslint warning

```suggestion
import type { EffortCurves, RollingStock, Comfort } from 'common/api/osrdEditoastApi';
```